package.json

```diff
@@ -56,5 +57,10 @@
   "bugs": {
     "url": "https://github.com/paldepind/snabbdom/issues"
   },
-  "homepage": "https://github.com/paldepind/snabbdom#readme"
+  "homepage": "https://github.com/paldepind/snabbdom#readme",
+  "husky": {
+      "hooks": {
+        "pre-commit": "npm test"
```

Comment:
Worked for me. Probably the most popular git hook helper on npm... I have some experience with it.